### DISCUSSION TITLE
--editable for manifest mode

### DISCUSSION THREAD

**Discussion by diehard2** [microsoft/vcpkg#36357] [Ideas]
I maintain an internal repository and its very difficult to hunt down issues with packages without --editable. I can do it in classic mode, but I can't install the necessary versions of dependencies. My search for installing a version of a dependency in classic mode came up blank. So I have to install the version I need in manifest mode, then copy it over to the classic vcpkg_installed folder.

All of this took awhile to figure out. It would greatly simplify internal repository maintenance if --editable was available in manifest mode.

**Comment by dg0yt** [comment]
Known as #16874.

**Comment by diehard2** [comment]
so, don't hold my breath that this is going to get any easier anytime soon, haha. thanks for the answer @dg0yt. Its not usually an issue unless I need to get a specific dependency version in to test against
